pkg/kubelet/rkt/rkt.go

```diff
@@ -1148,6 +1150,24 @@ func constructSyslogIdentifier(generateName string, podName string) string {
 	return podName
 }
 
+// Setup additional systemd field specified in the Pod Annotation
+func setupSystemdCustomFields(annotations map[string]string, unitOptionArray *[]*unit.UnitOption) error {
```

Comment:
nit: maybe return `([]unit.UnitOption, error)` rather than manipulating the pointers which is a less common style.